source/shared/cpp/ObjectModel/ParseUtil.h

```diff
@@ -184,4 +187,74 @@ namespace AdaptiveSharedNamespace
         T result = jsonObject.empty() ? defaultValue : deserializer(jsonObject, defaultValue);
         return result;
     }
+
+    template<typename T>
+    std::vector<std::shared_ptr<T>> ParseUtil::GetElementCollection(bool isTopToBottomContainer,
```

Comment:
So just continuing with improving unit test coverage (if we dont already have it) let's make sure this function has good unit test coverage

Reply:
We don't call this function directly from the tests, but the logic here is well tested by the ContainerStyleTest.cpp test cases.

---
In reply to: [279807494](https://github.com/Microsoft/AdaptiveCards/pull/2802#discussion_r279807494) [](ancestors = 279807494)